libs/cdk/src/lib/zone-configuration/zone-cfg.ts

```diff
@@ -84,80 +118,62 @@ function createZoneFlagsConfigurator(): ZoneConfig {
     });
   };
 
-  return {
+  const config = {
+    /**
+     * Unpatch all related to XHR
+     **/
+    unpatchXHR: () => {
+      config.global.disable.XHR();
+      config.events.disable.UNPATCHED_EVENTS([...xhrEvent]);
+    },
+    /**
+     * Unpatch passive events https://developers.google.com/web/updates/2016/06/passive-event-listeners
+     **/
+    usePassiveScrollEvents: () => {
```

Comment:
The naming here is not intuitive in terms of the additional unpatch event.

Reply:
done!
also reorganised code and interfaces here a bit, check it please

Reply:
👍